arrow-libs/core/arrow-core/src/commonTest/kotlin/arrow/core/IterableTest.kt

```diff
@@ -551,4 +553,87 @@ class IterableTest : StringSpec({
       }
     }
 
+  "unzip is the inverse of zip" {
+    checkAll(Arb.list(Arb.int())) { xs ->
+
+      val zipped = xs.zip(xs)
+      val ls = zipped.unzip()
+      val rs = xs to xs
+
+      ls shouldBe rs
+    }
+  }
+
+  "unzip(fn)" {
+    checkAll(Arb.list(Arb.pair(Arb.int(), Arb.string()))) { xs ->
+      xs.unzip { it } shouldBe xs.unzip()
+    }
+  }
+
+  "zip is the inverse of unzip" {
+    checkAll(Arb.list(Arb.pair(Arb.int(), Arb.string()))) { xs ->
+      val (l, r) = xs.unzip()
+
+      l.zip(r) shouldBe xs
+    }
+  }
```

Comment:
**Nit**: This is testing Kotlin Std `zip` method. I think this can be removed.

Reply:
done